test/integration/multikueue/suite_test.go

```diff
@@ -213,10 +214,17 @@ func managerAndMultiKueueSetup(ctx context.Context, mgr manager.Manager, gcInter
 	err := multikueue.SetupIndexer(ctx, mgr.GetFieldIndexer(), managersConfigNamespace.Name)
 	gomega.Expect(err).NotTo(gomega.HaveOccurred())
 
+	adapters, err := jobframework.GetMultiKueueAdapters(sets.New([]string{
+		"batch/job", "kubeflow.org/mpijob", "ray.io/rayjob", "ray.io/raycluster",
+		"jobset.x-k8s.io/jobset", "kubeflow.org/mxjob", "kubeflow.org/paddlejob",
+		"kubeflow.org/pytorchjob", "kubeflow.org/tfjob", "kubeflow.org/xgboostjob"}...))
```

Comment:
```suggestion
	adapters, err := jobframework.GetMultiKueueAdapters(sets.New[string](
		"batch/job", "kubeflow.org/mpijob", "ray.io/rayjob", "ray.io/raycluster",
		"jobset.x-k8s.io/jobset", "kubeflow.org/mxjob", "kubeflow.org/paddlejob",
		"kubeflow.org/pytorchjob", "kubeflow.org/tfjob", "kubeflow.org/xgboostjob"))
```

Also the slice of enabled integrations  can be an argument  to this function,  whit this you can move the tests from "./failedsetup" , in this suite (but sure in their own `var _ = ginkgo.Describe(` )